crates/symbolicator-service/tests/integration/sourcemap.rs

```diff
@@ -36,6 +36,7 @@ fn make_js_request(
         stacktraces,
         modules: vec![],
         dist: dist.into(),
+        allow_scraping: true,
```

Comment:
actually, we can revert the "intentionally malformed url" if we just disallow scraping :-)